svelte.config.js

```diff
@@ -11,23 +8,13 @@ const config = {
 	preprocess: preprocess(),
 
 	kit: {
-		adapter: adapter(),
-
 		files: { assets: 'src/content' },
+		adapter: adapter()
+	},
 
-		// Override http methods in the Todo forms
-		methodOverride: {
-			allowed: ['PATCH', 'DELETE']
-		},
-
-		paths: {
-			base: process.env.SVELTE_BASE_PATH ?? '',
-		},
+	paths: {
+		base: process.env.SVELTE_BASE_PATH ?? '',
```

Comment:
This needs to be changed back to how it was (`config.kit.paths.base` was changed to `config.paths.base`).

If you follow the instructions in the [readme](https://github.com/etinaude/portfolio-workshop#readme) to setup GitHub Pages (it takes ~2 minutes), you'll see that currently images don't load properly.